docs/Configuration/fleet-server-configuration.md

```diff
@@ -2896,6 +2896,40 @@ The duration between DEP device syncing (fetching and setting of DEP profiles).
   mdm:
     apple_dep_sync_periodicity: 10m
   ```
+##### fleet_mdm_windows_wstep_identity_cert_bytes
+The content of the Windows WSTEP identity certificate. An X.509 certificate, PEM-encoded.
+- Default value: ""
+- Environment variable: fleet_mdm_windows_wstep_identity_cert_bytes  
+- Config file format: 
+  ```
+  mdm:
+   fleet_mdm_windows_wstep_identity_cert_bytes: |
+      -----BEGIN CERTIFICATE-----
+      ... PEM-encoded content ...
+      -----END CERTIFICATE-----
+  ```
+
+
+
+> Windows MDM features are not ready for production and are currently in development. These features are disabled by default. Generated when a user turns off MDM features for all Windows hosts.
```

Comment:
```suggestion
> Windows MDM features are not ready for production and are currently in development. These features are disabled by default.
```

Where did the last sentence come from? I don't think it's relevant. Could be wrong

Reply:
Also, I think we should move this purple notice box up so it sits right below the `mdm.windows_wstep_identity_cert_bytes` title. This way, the user sees this important notice first before they read on about the config option.